File: beginner/pagina10/2770/2770.py
while True:
    try:
        larguraPlaca, alturaPlaca, qntPedidos = map(int, input().split())

        areaPlca = larguraPlaca * alturaPlaca

        for _ in range(qntPedidos):
            larguraPdC, alturaPdc = map(int, input().split())

            areaPlacaPdc = alturaPdc * larguraPdC

            if areaPlacaPdc <= areaPlca and \ 
                ((larguraPdC <= larguraPlaca) and (alturaPdc <= alturaPlaca) or 
                (larguraPdC <= alturaPlaca and alturaPdc <= larguraPlaca)):
                print("Sim")
            else:
                print("Nao")

    except EOFError:
        break 

File: beginner/__2000__/2770/2770.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").trim().split('\n')

let index = 0;

while(index < input.length){

    const [larguraPlaca, alturaPlaca, qnt] = input[index++].split(' ').map(Number)
    const areaPlaca = larguraPlaca * alturaPlaca

    for(let i = 0; i < qnt; i++){
        const [larguraPdc, alturaPdc] = input[index++].split(' ').map(Number)
        
        const cabeNormal = larguraPdc <= alturaPlaca  && alturaPdc <= larguraPlaca
        const cabeRotacionado = larguraPdc <= larguraPlaca && alturaPdc <= alturaPlaca

        if(!cabeNormal && cabeRotacionado){
            console.log("Nao")
            continue
        }

        const areaPDc = larguraPdc * alturaPdc
        console.log(areaPDc <= areaPlaca ? "Sim" : "Nao")
    }
}
